### PR TITLE
Dac6 79

### PR DESCRIPTION
**Pull request by daniel-manning**
Add a back end microservice for HOD calls

### CODE REVIEW

File: test/base/SpecBase.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package base
+
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ */
```
Comment (helensoniega) [inline test/base/SpecBase.scala:22]: Remove multi-line comment

File: app/controllers/BusinessMatchingController.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                                          (implicit executionContext: ExecutionContext) extends
+  BackendController(controllerComponents) {
+
+  def individualMatchingSubmission(nino: Nino): Action[JsValue] = Action(parse.json).async {
+    implicit request =>
+      val individualMatchingSubmissionResult: JsResult[IndividualMatchingSubmission] =
+        request.body.validate[IndividualMatchingSubmission]
+
+      individualMatchingSubmissionResult.fold(
+        invalid = _ => Future.successful(BadRequest("")),
```
Comment (helensoniega) [inline app/controllers/BusinessMatchingController.scala:42]: Provide a message about what caused BadRequest?

**Review by helensoniega — APPROVED** [review]

**Review by helensoniega — APPROVED** [review]

File: test/base/SpecBase.scala (hunk 1 above)
Comment (daniel-manning) [reply test/base/SpecBase.scala:22]: Automated Copyright header comes from it being an open repo

File: app/controllers/BusinessMatchingController.scala (hunk 2 above)
Comment (daniel-manning) [reply app/controllers/BusinessMatchingController.scala:42]: I'm not sure we want to be passing up the validation results. The caller will check their submission.

**Pull request merged**